### DISCUSSION TITLE
Order extensions by date

### DISCUSSION THREAD

**Discussion by AlexD81** [Comfy-Org/ComfyUI-Manager#325] [General]
Hi, is there a possibility to order extensions by date?   Or is it possible to add it?     It's really important,  as we have 450 extensions already, and it's  hard to track new ones

**Comment by ltdrdata** [comment]
Newly added nodes can be viewed by switching to the "recent" channel.
![image](https://github.com/ltdrdata/ComfyUI-Manager/assets/128333288/ecec1f15-6ff6-4a48-b6f0-2e645d4fb529)

There are plans to add a date sorting feature in the future.
It's a fairly labor-intensive task, so it might take some time.

And since the date sorting will include updates, new nodes still need to be checked in the "recent" channel.
